### PR TITLE
feat(handlers): add kickFromVoiceChannel() function

### PR DESCRIPTION
**Pull request by itohatweb**
**Please describe the changes this PR makes and why it should be merged:**

**Status**

- [x] Code changes have been tested against the Discord API, or there are no code changes

**Semantic versioning classification:**

- [x] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Review by ayntee — APPROVED** [review]
LGTM ― maybe a predictable name would be appreciated.

**Comment by itohatweb** [comment]
eg. `disconnectMemberFromVoiceChannel`, `kickMemberFromVoiceChannel`?

File: src/api/handlers/member.ts [hunk 1]
```diff
@@ -240,6 +240,11 @@ export function moveMember(
   return editMember(guildID, memberID, { channel_id: channelID });
 }
 
+/** Kicks a member from a voice channel */
+export function kickFromVoiceChannel(guildID: string, memberID: string) {
```
Comment (Skillz4Killz) [inline src/api/handlers/member.ts:244]: Is it possible to add getters for this on some of the structs? For example, `channel.kickFromVoice()`, `member.kickFromVoice()`, `guild.kickFromVoice()`

Comment (itohatweb) [reply src/api/handlers/member.ts:244]: MilSci has already added getters. He is waiting for this PR to be merged.

**Pull request merged**